app.rb

```diff
@@ -0,0 +1,82 @@
+require_relative './book'
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+
+class App
+  attr_reader :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_all_books
+    books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_all_people
+    people.each { |person| puts "[#{person.class}] Name: #{person.name} ID: #{person.id} age: #{person.age}" }
+  end
+
+  def add_teacher
```

Comment:
- [ ] **REQUIRED:** It will be nice to have one method like a create person method that will create either a student or a teacher in a way to keep your code DRY (Do not Repeat Yourself)